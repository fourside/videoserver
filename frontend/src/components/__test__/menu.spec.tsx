import * as React from 'react';
import 'jest';
import * as renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

import Menu from '../menu';

test('snapshot', () => {
  const component = renderer
    .create(
      <MemoryRouter>
        <Menu toggleModal={() => {}} />
      </MemoryRouter>
    )
    .toJSON();
  expect(component).toMatchSnapshot();
});
