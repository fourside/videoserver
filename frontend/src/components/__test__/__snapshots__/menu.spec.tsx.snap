// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className=" dropdown is-right"
>
  <div
    className="dropdown-trigger"
  >
    <button
      aria-controls="dropdown-menu"
      aria-haspopup="true"
      className="button"
      onClick={[Function]}
    >
      <span>
        menu
      </span>
      <span
        className="icon is-small"
      >
        <i
          aria-hidden="true"
          className="fas fa-bars"
        />
      </span>
    </button>
  </div>
  <div
    className="dropdown-menu"
    id="dropdown-menu"
    role="menu"
  >
    <div
      className="dropdown-content"
    >
      <a
        aria-current="page"
        className="dropdown-item active"
        href="/"
        onClick={[Function]}
        style={Object {}}
      >
        Home
      </a>
      <a
        className="dropdown-item"
        onClick={[Function]}
      >
        Form
      </a>
      <a
        aria-current={null}
        className="dropdown-item"
        href="/list"
        onClick={[Function]}
      >
        List
      </a>
      <a
        className="dropdown-item"
        href="/feed"
      >
        RSS
      </a>
    </div>
  </div>
</div>
`;
